contracts/token/StakedElyfiToken.sol

```diff
@@ -5,13 +5,36 @@ import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
 import '@openzeppelin/contracts/token/ERC20/ERC20.sol';
 import '@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol';
 import '@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol';
+import '../libraries/ERC20Metadata.sol';
 
 contract StakedElyfiToken is ERC20, ERC20Permit, ERC20Votes {
```

Comment:
This should be renamed to `StakedToken`, but it makes hard to see what is changed in this PR.
I will rename the contract after the review.